packages/protocol/contracts/identity/Attestations.sol

```diff
@@ -351,6 +411,26 @@ contract Attestations is
     setWalletAddress(walletAddress);
   }
 
+  /**
+   * @notice Returns the current active attestation request for an identifier/account pair, if any
+   * @param identifier Hash of the identifier.
+   * @param account Address of the account
+   * @return [Block numbr at which was requested, Number of requests]
+   */
+  function getActiveAttestationRequest(
```

Comment:
What does `active` mean in this context? Is an attestation request still `active` if it has been `attestationExpiryBlocks` since the request?

Reply:
Active in this context means that it is a request for attestation for which issuers were not selected yet. Maybe a better word would be `getAttestationRequest` or `getUnselectedAttestationRequest`? I'll change it to the former for now.